mm2src/coins/rpc_command/lightning/mod.rs

```diff
@@ -0,0 +1,12 @@
+pub mod close_channel;
+pub mod connect_to_node;
+pub mod generate_invoice;
+pub mod get_channel_details;
+pub mod get_claimable_balances;
+pub mod get_payment_details;
+pub mod list_channels;
+pub mod list_payments_by_filter;
+pub mod open_channel;
+pub mod send_payment;
+pub mod trusted_nodes;
+pub mod update_channel;
```

Comment:
Can we group this modules? Like re-exporting all the payment related ones from pub mod payment{}, channel related ones from pub mod channel, etc

Reply:
I like this idea :)
I can even add a namespace for `channels::`, `nodes::`, `payments::` to the RPC methods names